.travis.yml

```diff
@@ -1,11 +1,15 @@
+dist: trusty
```

Comment:
Note from our conversation: Travis doesn't yet support builds on Xenial, and it doesn't seem to be a high priority for them right now.